source/includes/_request-dictionary.md.erb

```diff
@@ -376,7 +376,7 @@ Name | Description
 </p>
 
 <p>
-  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddargetter-how">
+  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddar-how">
```

Comment:
To be clear, these need to be cross-referenced with what currently exists and make a change only as necessary. For example, this one was actually already changed back in the day to:

http://support.cheddargetter.com/kb/getting-started-19/customer-data-migration-and-import